src/suggestion.rs

```diff
@@ -135,6 +145,67 @@ impl<'s> fmt::Display for Suggestion<'s> {
             0
         };
 
+        // We will trim the literal if does not fit in the terminal size
+        // The misspelled word shall always be shown and we aim to show as much
+        // info as possible to the user easily locate the word
+        let terminal_size: usize = get_terminal_size();
+        let mut max_chars: usize = terminal_size;
+        let mut min_chars = 0usize;
+        // The paddings give some space for the ` {} ...` and extra indentation and formatting:
+        // ```
+        // 65 |  ...  here, on the second most pointless site ever! Well, wander ont  ...
+        //    |                                                                  ^^^
+        //    |                                                                                                                                                                                                ^^^
+        //    | - not, Ont, int, on, onto, cont, font, or one of 9 others
+        //    |
+        //    |   Possible spelling mistake found.
+        //    |
+        // ```
+        let padding_offset = 5;
+        let padding_end = 15usize;
+
+        let range_start_text = Range {
+            start: 0usize,
+            end: marker_range_relative.start,
+        };
+        let range_end_text = Range {
+            start: marker_range_relative.end,
+            end: self.literal.as_str().chars().count(),
```

Comment:
try using `self.rendered` (don't remember the exact name, or `self.content`), `literal` has potentially spare characters, it's not a good idea to roll with that, the characters are not of fixed length.

Reply:
I could not find it in `Suggestion`. Maybe belongs to another struct?

Reply:
Ah, yes, those changes just landed in master yesterday (quite some significant refactor) so in `master` `Suggestion` contains `CheckableChunk` as`chunk` instead of `literal` - but I'll take care of the rebase once this PR is ready

Reply:
Thanks for doing the rebase. Let me know if you need any help with the conflicts.